HTMLSerializer.js

```diff
@@ -38,6 +38,20 @@ var HTMLSerializer = class {
     ]);
 
     /**
+     * @private {Object<string, string>} The keys are all characters that need
+     *     to be properly escaped when in a text node.  The value is the
+     *     properly escaped string.
+     * @const
+     */
+    this.escapedCharacters = {
```

Comment:
Since this is a constant, can you use angry caps? It might be useful to add "map" too, like CHARACTER_ESCAPING_MAP.